Code/Menu.cs

```diff
@@ -0,0 +1,35 @@
+using Godot;
+using System;
+
+using Godot.Collections;
+
+public partial class Menu : Node2D
```

Comment:
I should make this its own godot package or something. Going to be doing this in other projects for sure

Reply:
mmm, actually a decent bit of this is "blueprint" specific. I just need to make a note of the first 2 lines. annoying to re-learn the correct casting